packages/changed-elements-react/README.md

```diff
@@ -33,14 +35,69 @@ To begin using this package in your application, you will need to:
     });
     ```
 
-4. Mount `<VersionCompareSelectComponent />` to begin version comparison worflow by selecting iModel versions to compare. You should also add a button in your UI that calls `VersionCompare.manager.stopComparison()`.
-5. `<ChangedElementsWidget />` React component lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility.
+4. `<ChangedElementsWidget />` React component lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility. The following code has example of widget use but does not show off all props for widget.
+
+    ```ts
+    import { ChangedElementsWidget } from "@itwin/changed-elements-react";
+    <ChangedElementsWidget
+          useV2Widget
+          feedbackUrl="https://example.com"
+          iModelConnection={UiFramework.getIModelConnection()!}
+          enableComparisonJobUpdateToasts
+          manageNamedVersionsSlot={<ManageNamedVersions />}
+          documentationHref="https://example.com"
+        />,
+    ```
+
+5. `<NamedVersionSelectorWidget />` **Experimental** React component lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility. The following code has example of widget use but does not show off all props for widget.
+
+    ```ts
+    import { NamedVersionSelectorWidget } from "@itwin/changed-elements-react";
+    <NamedVersionSelectorWidget
+      iModel={iModel}
+      manager={VersionCompare.manager}
+      manageVersions={<ManageNamedVersions />}
+      feedbackUrl="https://example.com"
+      documentationHref="https://example.com"
+      />
```

Comment:
You have inconsistent tab-spacing in each of your examples. Make them all consistent.